File: Java/1043 - Triângulo.java
/* 
   ==UserScript==
 @name         Triângulo
 @namespace    https://github.com/Ddavidi/uri-online-judge-beecrowd
 @description  URI 1043 - Triângulo
 @copyright    @ddavidi_
 @author       @ddavidi_	
   ==/UserScript==
*/

import java.util.Scanner;
import java.io.IOException;
import java.util.stream.*;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        // Scanner
        Scanner scan = new Scanner(System.in);
        
        double a = scan.nextDouble();
        double b = scan.nextDouble();
        double c = scan.nextDouble();
        
        if(a < b+c && b < a+c && c < a+b){
            double perimetro = a+b+c;
            System.out.println(String.format("Perimetro = %.1f", perimetro));
        }
        else{
            double area = (a+b)*c/2;
            System.out.println(String.format("Area = %.1f", area));
        }
        
    }
}   

File: C/1043 - Triângulo.c
/* 
   ==UserScript==
 @name         Triângulo
 @namespace    https://github.com/Ddavidi/uri-online-judge-beecrowd
 @description  URI 1043 - Triângulo
 @copyright    @ddavidi_
 @author       @ddavidi_	
   ==/UserScript==
*/

#include <stdio.h>
#include <math.h>

int main()
{
    double A, B, C, Perimetro, Area;
    
    scanf("%lf%lf%lf", &A, &B, &C);
    
    if(A>B-C && A<B+C && B>A-C && B<A+C && C>A-B && C<A+B){
        Perimetro = A+B+C;
        printf("Perimetro = %.1lf\n", Perimetro);
    }
    else{
        Area = (A+B)*C/2;
        printf("Area = %.1lf\n", Area);
    }
    
    return 0;
}
